src/Rendering.hs

```diff
@@ -11,29 +11,37 @@ module Rendering where
 
   -- * make a wall
```

Comment:
Doesn't this create a Haddock section ? You probably want to use `-- |` instead.